articles/guides/login/centralized-vs-embedded.md

```diff
@@ -0,0 +1,79 @@
+:::
+
+In this article, we will evaluate the pros and cons of these two options and see how you can use Auth0 to implement them.
+
+## Pros and cons
+
+- **Single Sign-On (SSO)**: If you are working with mobile apps you cannot have SSO unless you use centralized login. With web apps you can, although the most secure way is to use a central service so the cookies are from the same origin. With embedded login, you'd have to collect the user credentials in an application served from one origin and then send them to another origin, which can present certain security vulnerabilities, including the possibility of a phishing attack (see [Embedded Login with Auth0 > Security risks](#security-risks) for more info). There are workarounds you could use, like third-party cookies, but the most secure option for SSO, and logins in general, is using a central service.
+
+- **Consistency and Maintenance**: With embedded login, if you have more than one app, you will have to implement more than one login page. You will also have to maintain and manage these pages. Besides the extra effort it can also introduce inconsistencies which results in bad UX. Furthermore, with embedded login you would have to manage the dangers of cross-origin attack vectors. On the other hand, if you are using centralized login, then your Authorization Server (the domain that logs the users in) owns all the login pages which makes the management easier and the pages more consistent and secure. You could also use a single login page among your apps, a process that creates an impression that users are logging into a centralized system, rather than an individual app. In the following diagram you can see an example of how the centralized and embedded logins look like. The reason why the centralized login offers a more consistent and thus superior use experience is evident.
+
+![Centralized vs Embedded login UX](/media/articles/guides/login/centralized-embedded-ux.jpg)
+
+- **Central Features Management**: When you use centralized login with Auth0, you can turn on and off features across all your apps, using the Dashboard. An example is [Multifactor Authentication](/multifactor-authentication) which you can enable using the toggles located at the [Dashboard > Multifactor Auth](${manage_url}/#/guardian) page. These changes will be automatically available to all your registered apps.
+
+- **User Experience**: In the past, an argument could be made that the user experience with embedded login was better because it did not require redirecting users to another domain, which could be alerting. However, users are getting increasingly familiar with the process of being redirected to another domain to log in, mainly because they use social logins all the time. As a result, they don't find the process disruptive to their experience. Ask yourself, have you ever clicked **Login with Google** and get frustrated because you are redirected to Google's domain? You probably don't even notice it any longer.
```

Comment:
Should we mention custom domains here as well? I know it's mentioned later in the article, but someone reading this section might get the impression that centralized login = different domains.

Reply:
@thameera Good point, we mean subdomains. I will make some adjustments (and also refer to custom domains).